src/main/java/com/nojsoft/dao/UserDao.java

```diff
@@ -28,22 +28,28 @@ public User saveOrUpdate(User user) {
     }
 
     public List<User> getUsersByAccessKey(String accessKey) {
-        return super.findByField(User.class, DataBaseConstants.USER_ACCESS_KEY_FIELD, accessKey);
+        return super.findByField(User.class, DataBaseConstants.USER_EMAIL_FIELD, accessKey);
     }
 
-    public User getUserStatus(User user) {
-        List<User> users = super.findByField(User.class, DataBaseConstants.USER_TOKEN_FIELD, user.getToken());
-        user.setActive(!(users.isEmpty() || users.size() == 0));
+    public User getUserStatus(String userId) {
+        User user = null;
+        System.out.println("Method: "+userId);
```

Comment:
Pondria algo mas informativo en el log, algo como 
`System.out.println(this.getClass().getSimpleName() + "/getUserStatus - userId : " + userId);`
Seria bueno integrar Log4j para estandarizar todos los logs

Supongo java 8 tiene algo mas elegante para interpolacion de strings pero no la conozco.